go/proxyd/server.go

```diff
@@ -22,6 +24,7 @@ const (
 	ContextKeyAuth          = "authorization"
 	ContextKeyReqID         = "req_id"
 	ContextKeyXForwardedFor = "x_forwarded_for"
+	MaxBatchRPCCalls        = 25
```

Comment:
do we know the batch size of ChainLink requests? Just want to make sure that this is sufficient.

Reply:
I'll bump it to something higher, like 100.